services/distribution/src/main/java/app/coronawarn/server/services/distribution/dgc/DigitalGreenCertificateToProtobufMapping.java

```diff
@@ -67,7 +67,7 @@ VaccineJsonStringObject readProphylaxisJson() {
    *
    * @return the protobuf filled with values from JSON.
    */
-  public ValueSets constructProtobufMapping() {
+  public ValueSets constructProtobufMapping(String language) throws DefaultValuesetsMissingException {
```

Comment:
Parameter "String language" is not used. Maybe we could remove it then?

Reply:
Hmm. Currently we cannot discern the language because of the underlying data structures. But once this needs implementing, this is the place to go. Remove until then?

Reply:
Yeah, removed it...